WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -956,9 +956,6 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
         case WPAnalyticsStatMediaServiceUploadCanceled:
             eventName = @"media_service_upload_canceled";
             break;
-        case WPAnalyticsStatMenusAccessed:
```

Comment:
I didn't have a chance to do a complete review today but leaving a note here that we might be able to clean up `WPAnalyticsStatMenusCreatedItem`, `WPAnalyticsStatMenusCreatedMenu`, `WPAnalyticsStatMenusDeletedMenu`, `WPAnalyticsStatMenusDeletedItem`, `WPAnalyticsStatMenusDiscardedChanges`, `WPAnalyticsStatMenusEditedItem`, `WPAnalyticsStatMenusOpenedItemEditor`, and `WPAnalyticsStatMenusSavedMenu`

Reply:
Good catch. Removed with 33ef2aa3727b34220c854f4f2ed77b06405a310a